test/sharness/t0700-remotepin.sh

```diff
@@ -37,19 +38,56 @@ test_expect_success "test 'ipfs pin remote service ls'" '
   grep -q test_invalid_url_dns_svc ls_out
 '
 
-# SECURITY of access tokens in Api.Key fields:
-# Pinning.RemoteServices includes Api.Key, and we give it the same treatment
+test_expect_success "test enabling mfs pinning" '
+  ipfs config --json Pinning.RemoteServices.test_pin_mfs_svc.Policies.MFS.RepinInterval \"10s\" &&
+  ipfs config --json Pinning.RemoteServices.test_pin_mfs_svc.Policies.MFS.PinName \"mfs_test_pin\" &&
+  ipfs config --json Pinning.RemoteServices.test_pin_mfs_svc.Policies.MFS.Enable true &&
+  ipfs config --json Pinning.RemoteServices.test_pin_mfs_svc.Policies.MFS.RepinInterval > repin_interval &&
+  ipfs config --json Pinning.RemoteServices.test_pin_mfs_svc.Policies.MFS.PinName > pin_name &&
+  ipfs config --json Pinning.RemoteServices.test_pin_mfs_svc.Policies.MFS.Enable > enable &&
+  echo 10s > expected_repin_interval &&
+  echo mfs_test_pin > expected_pin_name &&
+  echo true > expected_enable &&
+  test_cmp repin_interval expected_repin_interval &&
+  test_cmp pin_name expected_pin_name &&
+  test_cmp enable expected_enable
+'
+
+# expect PIN to be created
+test_expect_success "verify MFS root is being pinned" '
+  ipfs files cp /ipfs/bafkqaaa /mfs-pinning-test-$(date +%s.%N) &&
+  ipfs files flush &&
+  sleep 31 &&
+  ipfs files stat / --enc=json | jq -r .Hash > mfs_cid &&
+  ipfs pin remote ls --service=test_pin_mfs_svc --name=mfs_test_pin --status=queued,pinning,pinned,failed --enc=json | tee ls_out | jq -r .Cid > pin_cid &&
+  cat mfs_cid ls_out &&
```

Comment:
intentional or just for debugging? (same below)

Reply:
Intentional, this output is very useful when ` -v` is passed while running sharness to identify cause of regression.